lectures/overview.js

```diff
@@ -300,7 +300,7 @@ prose(
   'In the 1980s, there was a renewed interest in neural networks.',
   'Backpropagation was rediscovered and popularized as a way to actually train deep neural networks,',
   'and Yann LeCun built a system based on convolutional neural networks to recognize handwritten digits,',
-  'one of first successful use of neural networks that became used by the USPS to recognize zip codes.',
+  'one of the first successful usages of neural networks that became used by the USPS to recognize zip codes.',
```

Comment:
rewrite: This was one of the first successful uses of neural networks, which was then deployed by the USPS to recognize zip codes.